### ISSUE TITLE
Spatie permission dependent ?

### ISSUE THREAD

**Issue by scramatte** [pktharindu/nova-permissions#71]
Hi,

Does your package is spatie permission dependent?
https://spatie.be/docs/laravel-permission/v5/introduction

Regards

**Comment by pktharindu** [comment]
No, it has no composer dependencies, other than Nova, of course.
